crates/common/types/transaction.rs

```diff
@@ -1594,7 +1594,7 @@ mod serde_impl {
         pub gas: Option<u64>,
         #[serde(default)]
         pub value: U256,
-        #[serde(default, with = "crate::serde_utils::bytes", alias = "data")]
+        #[serde(default, with = "crate::serde_utils::bytes")]
```

Comment:
Because when running the L2 we had this error with the alias on:
`Error: server returned an error response: error code -32000: Invalid params: duplicate field "input"`
So removing it fixed that for us. I still don't know why it happens though.

Reply:
This was added in this PR: https://github.com/lambdaclass/lambda_ethereum_rust/pull/915 to fix an issue with EIP1559 transactions that required data instead of input apparently. Unfortunately we don't have a test to check this regression. Maybe the error happened on a json with both input and data?